CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ install(FILES package.xml DESTINATION share/libnabo)
 export(PACKAGE libnabo)
 
 # Create variable with the library location
-get_property(libnabo_library TARGET ${LIB_NAME} PROPERTY LOCATION)
+#get_property(libnabo_library TARGET ${LIB_NAME} PROPERTY LOCATION)
```

Comment:
In case it's really not needed anymore, the code should be deleted and not commented.